src/Microsoft.CodeDom.Providers.DotNetCompilerPlatform/Util/CompilationSettings.cs

```diff
@@ -54,5 +80,14 @@ internal static class CompilationSettingsHelper {
             string compilerFullPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, relativePath);
             return compilerFullPath;
         }
+
+        private static bool IsDebuggerAttached {
+            get {
+                return IsDebuggerPresent() || Debugger.IsAttached;
+            }
+        }
+
+        [DllImport("kernel32.dll")]
```

Comment:
This project is for desktop version .net framework. If we decide to support Mono, we will change this code.